src/Sql/Sql.php

```diff
@@ -4,174 +4,121 @@
 
 use Laminas\Db\Adapter\AdapterInterface;
 use Laminas\Db\Adapter\Driver\StatementInterface;
-use Laminas\Db\Adapter\Platform\PlatformInterface;
+use Laminas\Db\Adapter\StatementContainerInterface;
```

Comment:
I think we still need
use Laminas\Db\Adapter\Platform\PlatformInterface;
see line 17

Reply:
actually looks like we just need
use Laminas\Db\Adapter\Platform; ?